testdata/p4_16_errors/table-entries-exact-ternary.p4

```diff
@@ -0,0 +1,86 @@
+/*
```

Comment:
If you name all these files ending with -bmv2, then the p4c-bm2-ss compiler will be used, and the output json will be validated by bmv2.